### PR TITLE
Fix for CVE-2021-4034 (PwnKit)

### PR DESCRIPTION
**Pull request by mkosola**

### CODE REVIEW

**Review by inzanity — APPROVED** [review]
Plugs the hole.

**Review by mlehtima — APPROVED** [review]

File: rpm/patches/debian/Local-Privilege-Escalation-in-polkit-s-pkexec-CVE-20.patch [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+--- a/src/programs/pkcheck.c
++++ b/src/programs/pkcheck.c
+@@ -353,6 +353,12 @@ main (int argc, char *argv[])
+   local_agent_handle = NULL;
+   ret = 126;
+ 
++  if (argc < 1)
++    {
++      usage (argc, argv);
++      exit(1);
```
Comment (rainemak) [inline rpm/patches/debian/Local-Privilege-Escalation-in-polkit-s-pkexec-CVE-20.patch:26]: Freedesktop returns here 126.

File: rpm/patches/debian/Local-Privilege-Escalation-in-polkit-s-pkexec-CVE-20.patch [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+ 
++
++  /*
++   * If 'pkexec' is called wrong, just show help and bail out.
++   */
++  if (argc<1)
++    {
++      clearenv();
++      usage (argc, argv);
++      exit(1);
```
Comment (rainemak) [inline rpm/patches/debian/Local-Privilege-Escalation-in-polkit-s-pkexec-CVE-20.patch:46]: Here 127

**Review by rainemak — APPROVED** [review]
Return values are a bit different depending of the distribution but at least this returns now. LGTM.

**Review by Thaodan — APPROVED** [review]

**Pull request merged**